test/runtests.jl

```diff
@@ -1,46 +1,10 @@
-using WangLandau
 using Test
-using Random
-
-include(joinpath(@__DIR__, "..", "examples", "ising.jl"))
+using WangLandau
 
 @testset "WangLandauProblem" begin
-    Random.seed!(12345)
-
-    # Define Problem
-    L = 5
-    periodic = false
-    statedefn = Ising2D(L; periodic)
-    prob = WangLandauProblem(statedefn)
-
-    @test WangLandau.histogram_size(statedefn) == (41,)
-    @test WangLandau.system_size(statedefn) == 25
-
-    state, index = WangLandau.initialise_state(statedefn)
-    @test state == [
-         1  -1   1  -1  -1;
-         1   1  -1  -1   1;
-        -1  -1  -1   1  -1;
-        -1  -1  -1   1   1;
-        -1   1   1  -1  -1
-    ]
-    @test index == 23
-
-    trial = random_trial!(state, prob.statedefn)
-    @test trial == CartesianIndex(4, 1)
-
-    new_index = histogram_index(state, statedefn, trial, index)
-    @test new_index == 26
-
-    commmit_state = WangLandau.commit_trial!(copy(state), statedefn, trial, index, new_index)
-    @test commmit_state == [
-        1  -1   1  -1  -1;
-        1   1  -1  -1   1;
-       -1  -1  -1   1  -1;
-        1  -1  -1   1   1;
-       -1   1   1  -1  -1
-    ]
+    include("problem.jl")
+end
```

Comment:
```suggestion
include("problem.jl")
```

there is already `@testset` within the file